services/addons/images/rsu_status_check/rsu_pinger.py

```diff
@@ -83,14 +84,10 @@ def run_rsu_pinger():
 
 
 if __name__ == "__main__":
-    # Configure logging based on ENV var or use default if not set
-    log_level = os.environ.get("LOGGING_LEVEL", "INFO")
-    log_level = "INFO" if log_level == "" else log_level
-    logging.basicConfig(format="%(levelname)s:%(message)s", level=log_level)
+    common_environment.configure_logging()
```

Comment:
similar question: is this implementation level "configure_logging" call needed since it's called already in the common library?

Reply:
I removed all configure_logging calls. The configure_logging runs automatically as long as the common_environment is imported somewhere, either directly or through one of the addon environment files. There were a few scripts which wouldn't have had any env var references, and for those I added a logging statement to print the logging level just to ensure the configure_logging would be called. Does that make sense?